test/services/return-requirements/submit-purpose.service.test.js

```diff
@@ -64,26 +65,18 @@ describe('Submit Purpose service', () => {
         Sinon.stub(PurposeValidation, 'go').resolves(null)
       })
 
-      it('fetches the current setup session record', async () => {
-        const result = await SubmitPurposeService.go(session.id, payload)
+      it('saves the submitted value', async () => {
+        await SubmitPurposeService.go(session.id, payload)
+
+        const refreshedSession = await session.$query()
 
-        expect(result.id).to.equal(session.id)
+        expect(refreshedSession.data.purpose).to.equal('Potable Water Supply - Direct')
       })
 
-      it('returns page data for the view', async () => {
+      it('returns page data for the journey', async () => {
```

Comment:
We didn't change this in `test/services/return-requirements/submit-no-returns-required.service.test.js`. And is it not more accurate to say we're returning page data for the view?

If 'view' isn't meaningful enough happy to discuss what we could use instead. But we probably need to then apply it in a separate PR so the tests remain consistent.

Reply:
I think where I'm coming from 'journey' represents all the pages the user went through. What is held in the session therefore is a better fit for 'data for the journey'.

When a `SubmitService` finds a validation error it will return data specifically for the Nunjucks view template (view), in this case `purpose.njk`.

On that basis, I'd stick with 'view'. If nothing else it avoids going back through all our other tests and updating them to use this term in place of 'view'.

Reply:
Again doh! Ignore me!